trie/patriciaMerkleTrie_test.go

```diff
@@ -1067,6 +1073,71 @@ func TestPatriciaMerkleTrie_ConcurrentOperations(t *testing.T) {
 	wg.Wait()
 }
 
+func TestPatriciaMerkleTrie_GetSerializedNodesClose(t *testing.T) {
+	t.Parallel()
+
+	args := getDefaultTrieStorageManagerParameters()
+	args.MainStorer = &storage.StorerStub{
+		GetCalled: func(key []byte) ([]byte, error) {
+			// gets take a long time
+			time.Sleep(time.Millisecond * 10)
+			return key, nil
+		},
+	}
+
+	trieStorageManager, _ := trie.NewTrieStorageManager(args)
+	tr, _ := trie.NewTrie(trieStorageManager, args.Marshalizer, args.Hasher, 5)
+	numGoRoutines := 1000
+	wgStart := sync.WaitGroup{}
+	wgStart.Add(numGoRoutines)
+	wgEnd := sync.WaitGroup{}
+	wgEnd.Add(numGoRoutines)
+
+	for i := 0; i < numGoRoutines; i++ {
+		if i%2 == 0 {
+			go func() {
+				time.Sleep(time.Millisecond * 100)
+				wgStart.Done()
+
+				_, _, _ = tr.GetSerializedNodes([]byte("dog"), 1024)
+				wgEnd.Done()
+				fmt.Println("finished")
```

Comment:
so we don't want to check the error here?

Reply:
forgotten prints :( 
No, not quite, the storer is a stub anyway. GetSerializedNodes will error as it won't find some hashes, GetSerializedNode would have returned what the mock storer returned